### DISCUSSION TITLE
Equivalent function of `transforms3d.euler.euler2quat`

### DISCUSSION THREAD

**Discussion by Jiayuan-Gu** [dfki-ric/pytransform3d#195] [Q&A]
Thx for the great library. I have not found an equivalent function of `transforms3d.euler.euler2quat` in pytransform3d. If there is not one, I would like to propose a feature request.

**Comment by AlexanderFabisch** [comment]
Hi @Jiayuan-Gu,

there is no such function yet. Feel free to open an issue. It's not a top priority to add more features to the library though. So if you want to get this done quickly, I would propose that you make a pull request. I could support with tests, documentation, etc.

**Reply by Jiayuan-Gu** [reply]
#196
